docs/designers-developers/developers/tutorials/block-tutorial/creating-dynamic-blocks.md

```diff
@@ -1,6 +1,15 @@
 # Creating dynamic blocks
 
-Dynamic blocks are blocks that can change their content even if the post is not saved. One example from WordPress itself is the latest posts block. This block will update everywhere it is used when a new post is published.
+Dynamic blocks are blocks that build their structure and content on the fly when the block is rendered on the front end.
+
+There are two primary uses for dynamic blocks:
+1. Blocks where content should change even if a post has not been updated. One example from WordPress itself is the latest posts block. This block will update everywhere it is used when a new post is published.
+2. Blocks where updates to the code (HTML, CSS, JS) should be immediately shown on the front end of the website. For example, if you update the HTML structure of a block by adding a new class, adding a div, or changing the layout in any other way, if you want these changes to be applied immediately on all occurrences of that block across the site, a dynamic block should be used. (If a dynamic block is not used then when block code is updated Guterberg's [validation process](https://developer.wordpress.org/block-editor/developers/block-api/block-edit-save/#validation) applies, causing users to see the validation message, "This block appears to have been modified externally").
```

Comment:
Would recommend not including the note around validation. This is a common occurrence but there are ways around it with block transforms (which should be better documented, admittedly).

```suggestion
2. Blocks where updates to the code (HTML, CSS, JS) should be immediately shown on the front end of the website. For example, if you update the structure of a block by adding a new class, adding an HTML element, or changing the layout in any other way, using a dynamic block ensures those changes are live across your entire site at the same time.
```